src/official-account/official-account.ts

```diff
@@ -299,9 +299,10 @@ class OfficialAccount extends EventEmitter {
   async sendFile (args: {
     file    : FileBox,
     touser  : string,
-    msgtype : OAMediaType
+    msgtype : OAMediaType,
   }): Promise<string> {
-    log.verbose('OfficialAccount', 'sendFile(%s)', JSON.stringify(args))
+    // log.verbose('OfficialAccount', 'sendFile(%s)', JSON.stringify(args))
```

Comment:
Please keep this log verbose output, because we always log a function when it's being called.

Reply:
ok, i uncommented it. just to note that this line does not work for audio file such as .mp3, and will throw an error.